### DISCUSSION TITLE
IDEA: disable email confirmation

### DISCUSSION THREAD

**Discussion by pmatuszy** [healthchecks/healthchecks#764] [Ideas]
Hi 
I've added a couple of email integrations and had to confirm the address for each check. What I would suggest:
1. disable email address confirmation whatsoever 
2. add email confirmation to the user profile so once it is confirmed it could be added to checks without confirmation
Paul

**Comment by cuu508** [comment]
You can disable email confirmation globally via [EMAIL_USE_VERIFICATION](https://healthchecks.io/docs/self_hosted_configuration/#EMAIL_USE_VERIFICATION).

If you are adding an email integration for your own email address (the precise same address that your user account is registered to), it should skip the email confirmation step too.
